gateway-ha/src/main/java/io/trino/gateway/ha/router/RoutingManager.java

```diff
@@ -66,7 +66,7 @@ public String load(String queryId)
                                     }
                                 });
 
-        this.backendToHealth = new ConcurrentHashMap<String, Boolean>();
+        this.backendToHealth = new ConcurrentHashMap();
```

Comment:
Avoid raw usage of parameterized map.

Reply:
is it necessary to put `<String, BackendHealthState>` or just `<>()` is enough?

Reply:
`<>()` is enough.